templates/install/macos_details.md

```diff
@@ -15,17 +15,13 @@ all users.
 Installing `elan` and mathlib supporting tools
 ---
 
-1.  Install [Homebrew](https://brew.sh/) if you do not already have it installed.
+1.  Grab the [latest release](https://github.com/leanprover/elan/releases/latest) for your architecture,
+    unpack it, and run the contained installation program.
 
-2.  Run `brew install elan-init` in a terminal window to
-    install `elan`.
+    The installation will tell you where it will install elan to (~/.elan by default),
+    and also ask you about editing your shell config to extend PATH. elan can be uninstalled via elan self uninstall, which should revert these changes.
```

Comment:
```suggestion
    The installation will tell you where it will install `elan` to (`~/.elan` by default),
    and also ask you about editing your shell config to extend `PATH`. `elan` can be uninstalled via `elan self uninstall`, which should revert these changes.
    
(We discourage using the `homebrew` provided `elan-init` package, as users often find that this lags behind updates to the official release. We especially discourage using the `homebrew` formula named simply `lean`, which installs a fixed version of Lean.)
```